x-pack/solutions/observability/plugins/infra/public/components/asset_details/header/host_header_title.tsx

```diff
@@ -18,7 +18,7 @@ export const HostHeaderTitle = ({
   title?: string;
   schema?: DataSchemaFormat | null;
 }) => {
-  return schema === DataSchemaFormat.SEMCONV ? (
+  return schema === 'semconv' ? (
```

Comment:
Can we use the `DataSchemaFormatEnum` here?

Reply:
`schema` is already typed with `DataSchemaFormat`. It will only accept either `semconv` or `ecs`.